0001/README.md

```diff
@@ -0,0 +1,396 @@
+---
+id: 1
+title: Call for discussion HOWTO
+status: discussion
+author:
+  - Gianluca Arbezzano <gianarb92@gmail.com>
+---
+
+## Summary
+
+_(In between parenthesis I will write what the section is for as a description.
+Summary contains a short summary of the CFD. Ideally it has to be useful for
+people who do not know what the CFD is to evaluate if they can be helpful or
+not, and what is the discussion about)_
+
+This is the first Request For Discussion (CFD) and it covers:
+
+*   What they are and why we need them
+*   How to write one
+*   When to avoid a CFD
+*   Lifecycle
+
+Rename the [proposal](https://github.com/tinkerbell/proposals) repository to CFD.
+
+## Goals and not Goals
+
+_(Goals and not Goals contains two bullet point sections that briefly describe
+the end goal for what you are discussion and what it is not covered from this
+discussion)_
+
+Goals
+
+* Explain what a CFD is, how and when to use it
+* Share the CFD lifecycle
+* Be the first CFD for the project
+
+Not Goals
+
+* Off all the automation, scripts and bots described as part of the CFD
+  lifecycle
+* Convert the already open PR against the
+    [proposal](https://github.com/tinkerbell/proposals) repo to this new format
+
+## Content
+
+_(the main section, it contains the core of the discussion, the design you are
+proposing, the new feature you are asking for and how it will be implemented)_
+
+### Why a CFD is needed and how it looks like
+
+The process of taking a decision about how and why a feature or a piece of code
+has to be written or not, and how is an important part of the engineering
+process.
+
+As an open source project Tinkerbell needs to have a public place where
+community members, contributors can share their ideas in detail, asking for
+help, feedback and more in general to spot problems addressable as early as
+possible.
+
+A maintainer can ask contributors to write a CFD in order to open a discussion
+about a feature that looks trivial, helping the contributors themself to sit
+down and think about the design of the request or the code they will build. It
+is a way to translate Slack discussions to something that won’t get lost for
+example.
+
+Currently we have a repo called
+[proposal](https://github.com/tinkerbell/proposals) that I would like to rename
+to CFD. Every CFD will have its own directory with a markdown file inside.
+
+### Metadata
+
+The CFD is written in markdown and it has a few metadata to place as header in
+the CFD file:
+
+* `ID`: identifies the RFD
+* `Title`: title of the RFD
+* `Status`: description of the current state for the RFD
+* `Authors`: in the format of a list of `Name <email@emai.com>`
+
+Here a complete example:
+
+```yaml
+---
+id: 1
+title: Call for discussion duty
+status: ideation
+authors: \
```

Comment:
There is some discrepancy between `authors` in this example, `author` in the [metadata for this CFD](https://github.com/tinkerbell/proposals/pull/2/files#diff-f24504494b88bfe6c96cd7617c38ce08R5), and `authors` (not a yaml array) [below](https://github.com/tinkerbell/proposals/pull/2/files#diff-f24504494b88bfe6c96cd7617c38ce08R109).